src/framework/ui/tab/tab.component.tsx

```diff
@@ -41,38 +39,32 @@ export class Tab extends React.Component<Props> {
         justifyContent: 'center',
         alignItems: 'center',
       },
-      icon: {},
+      icon: source.icon,
       title: source.text,
     };
   };
 
-  private createTextComponent = (style: StyleType): React.ReactElement<TextProps> => {
-    return (
+  private createTextComponent = (style: StyleType): React.ReactElement<TextProps> | null => {
+    return this.props.title ? (
```

Comment:
const hasTitle: boolean = this.props.title !== undefined;

return hasTitle ? this.renderTitle() : undefined;

Reply:
Don't like this. Needs discussion.